pkg/ccl/changefeedccl/kvfeed/physical_kv_feed.go

```diff
@@ -99,12 +99,17 @@ func (p rangefeedFactory) Run(ctx context.Context, sink kvevent.Writer, cfg rang
 	if cfg.RangeObserver != nil {
 		rfOpts = append(rfOpts, kvcoord.WithRangeObserver(cfg.RangeObserver))
 	}
-	rfOpts = append(rfOpts, kvcoord.WithConsumerID(cfg.ConsumerID))
+	if cfg.ConsumerID != 0 {
+		rfOpts = append(rfOpts, kvcoord.WithConsumerID(cfg.ConsumerID))
+	}
```

Comment:
this change seems unrelated. what's the purpose?

Reply:
Yeah it's not, I'm just cleaning it up to avoid adding an extra functional option when we don't need to.